### ISSUE TITLE
Find lefthook dependencies throws an error

### ISSUE THREAD

**Issue by mskri** [webpro/knip#325]
Using knip@2.38.4 with node 20.9.0.

Getting the following error when trying to run knip in a monorepo where I'm using lefthook. Downgrading to the previous version I used (2.33.3) fixes the issue temporarily.

```
Analyzing workspace ....
node:internal/fs/sync:25
  return binding.readFileUtf8(path, stringToFlags(flag));
                 ^

Error: ENOENT: no such file or directory, open '__FAKE__'
    at Object.readFileUtf8 (node:internal/fs/sync:25:18)
    at readFileSync (node:fs:441:19)
    at Module.findLefthookDependencies (file:///Users/<redacted>/node_modules/.pnpm/knip@2.38.4/node_modules/knip/dist/plugins/lefthook/index.js:26:20)
    at WorkspaceWorker.findDependenciesByPlugins (file:///Users/<redacted>/node_modules/.pnpm/knip@2.38.4/node_modules/knip/dist/WorkspaceWorker.js:191:59)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async WorkspaceWorker.findAllDependencies (file:///Users/<redacted>/node_modules/.pnpm/knip@2.38.4/node_modules/knip/dist/WorkspaceWorker.js:216:9)
    at async main (file:///Users/<redacted>/node_modules/.pnpm/knip@2.38.4/node_modules/knip/dist/index.js:109:30)
    at async run (file:///Users/<redacted>/node_modules/.pnpm/knip@2.38.4/node_modules/knip/dist/cli.js:24:73)
    at async file:///Users/<redacted>/node_modules/.pnpm/knip@2.38.4/node_modules/knip/dist/cli.js:74:1 {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '__FAKE__'
}
```

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.38.5. See [Release 2.38.5](https://github.com/webpro/knip/releases/tag/2.38.5) for release notes._

**Comment by webpro** [comment]
Thanks for reporting this @mskri! Should be OK now.
